File: 1st semester/BhaskaraFormula.java
import javax.swing.JOptionPane;

public class Main {
    public static void main(String[] args) {
        double a = Double.parseDouble(JOptionPane.showInputDialog("Insira o valor de a"));
        double b = Double.parseDouble(JOptionPane.showInputDialog("Insira o valor de b"));
        double c = Double.parseDouble(JOptionPane.showInputDialog("Insira o valor de c"));
        
        double delta = Math.pow(b, 2) - 4 * a * c;
        JOptionPane.showMessageDialog(null, "O valor de delta é " + delta);

        if (delta < 0) {
            JOptionPane.showMessageDialog(null, "Não existem raízes reais");
        } else if (delta == 0) {
            double x = -b / (2 * a);
            JOptionPane.showMessageDialog(null, "Só há uma raiz real, cujo valor é " + x);
        } else {
            double x1 = (-b + Math.sqrt(delta)) / (2 * a);
            double x2 = (-b - Math.sqrt(delta)) / (2 * a);
            JOptionPane.showMessageDialog(null, "Os valores das raízes reais são iguais à " + x1 + " e à " + x2);
        }
    }
}

File: 1st semester/BhaskaraFormula.py
import math
a = float(input("Insira o valor de a "))
b = float(input("Insira o valor de b "))
c = float(input("Insira o valor de c "))

delta = math.pow(b, 2) - 4 * a * c
print(f"O valor de delta é {delta:.3f}")

if delta < 0:
    print("Não existem raízes reais")
elif delta == 0:
    x = -b / (2*a)
    print(f"Só há uma raiz real, cujo valor é {x:.3f}")
else:
    x1 = (-b + math.sqrt(delta)) / (2*a)
    x2 = (-b - math.sqrt(delta)) / (2*a)
    print(f"Os valores das raízes reais são iguais à {x1:.3f} e à {x2:.3f}")
